maxim/logger/livekit/agent_session.py

```
@@ -31,35 +31,55 @@ def intercept_session_start(self: AgentSession, *args, **kwargs):
+    if room_id is not None:
+        session.add_tag("room_id", room_id)
+    if room_name is not None:
+        session.add_tag("room_name", room_name)
+    if session_id is not None:
+        session.add_tag("session_id", session_id)
+    if agent is not None:
+        session.add_tag("agent_id", str(id(agent)))
     # If callback is set, emit the session started event
     if get_livekit_callback() is not None:
         get_livekit_callback()(
             "maxim.session.started", {"session_id": session_id, "session": session}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)